dev/raytracing/shaders/fragment.glsl

```diff
@@ -89,8 +89,19 @@ uniform float lightBias;
 uniform float lightFalloff;
 uniform float brightness;
 
+const int num_tets = ##num_tets##;
```

Comment:
I wonder whether it is more natural to drop this and use ##num_tets## below.
I am actually considering of generating a code snipped "const int foo = VALUE;" from the compile time constant dictionary instead of doing string replacement. Any opinion?

For now, just leave as is.